### ISSUE TITLE
MISP.showorg

### ISSUE THREAD

**Issue by Grem25** [MISP/MISP#3430]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug, Question,
| OS version (server)       | Debian, ubuntu,
| OS version (client)       |  Mint
| PHP version               |7.0
| MISP version / git hash   | 2.4.XX, hash of the commit

### Expected behavior
FROM book:
The operator of the MISP instance has the possibility to hide the list of Organisations (enabled by the option “MISP.showorg”) ensuring anonymity of all recipients (i.e. organisations) in a community.

### Actual behavior
In fact it hides only the Orgc and Org of events but all user can see this list even without privileges. Global Actions -> Organisations

**Comment by iglocska** [comment]
Yeah this feature is super outdated and deprecated. I'll remove it completely.

**Comment by RichieB2B** [comment]
Or make ir admin only? Listing the local orgs is pretty handy.

**Comment by iglocska** [comment]
I mean it's completely unmaintaned, I am pretty sure there are ways around it but no one uses it and we've been telling people it's deprecated for the past 5 years or so :)

**Comment by iglocska** [comment]
(I meant the showorg toggle, not the org list, that will stay ofc)

**Comment by Grem25** [comment]
Ok, then for the description of MISP.showorg it states:
_"Setting this setting to 'false' will hide all organisation names / logos."_
that's correct even you said it's now deprecated.

Actually to hide organisation list is it possible to set:
Security.hide_organisation_index_from_users 
but it only works for users that haven't yet contributed data (has it's written in Changelog.txt)

It would be great to clarify this description in the configuration menu because now we find the following statement.

_"Enabling this setting will block the organisation index from being visible to anyone besides site administrators on the current instance"_

**Comment by iglocska** [comment]
Indeed, good point!
